lib/src/Chip/Chip.tsx

```diff
@@ -1,15 +1,32 @@
-import React from "react";
 import classnames from "classnames";
+import React from "react";
 import "./chip.scss";
 
 export type ChipProps = JSX.IntrinsicElements["div"] & {
     /** Event triggered when the close button is clicked */
-    onClose: React.MouseEventHandler<HTMLButtonElement>;
+    onClose: (e: React.MouseEvent<HTMLButtonElement> | React.KeyboardEvent<HTMLDivElement>) => void;
 };
 
-export const Chip: React.FC<ChipProps> = React.forwardRef(({ onClose, ...props }: ChipProps, ref: React.ForwardedRef<HTMLDivElement>) => (
-    <div {...props} ref={ref} className={classnames("rc", "chip", props.className)}>
-        <div className="content">{props.children}</div>
-        <button onClick={onClose}>&#x2715;</button>
-    </div>
-));
+export const Chip: React.FC<ChipProps> = React.forwardRef(({ onClose, ...props }: ChipProps, ref: React.ForwardedRef<HTMLDivElement>) => {
+    const onKeyDown = React.useCallback(
+        (event: React.KeyboardEvent<HTMLDivElement>) => {
+            switch (event.key) {
+                case "Backspace":
+                case "Delete": {
+                    onClose(event);
+                    break;
+                }
+            }
+        },
+        [onClose]
+    );
+
+    return (
+        <div {...props} ref={ref} role="button" className={classnames("rc", "chip", props.className)} onKeyDown={onKeyDown} tabIndex={0}>
```

Comment:
Maybe can look into this https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex#accessibility_concerns

Reply:
I was worried that changing the wrapper to button would be weird as one of the children is also a button, is it okay to nest button within button semantically?